Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/ScenarioCard.js

```diff
@@ -34,11 +34,11 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 ScenarioCard.propTypes = {
-    id: PropTypes.number.isRequired,
-    scenarioName: PropTypes.string.isRequired,
-    className: PropTypes.string.isRequired,
-    scenarioData: PropTypes.any.isRequired,
-    finish: PropTypes.any.isRequired,
+    id: PropTypes.string,
```

Comment:
why is ID a string not an int?

Reply:
I'm not sure I think that is how it was used?